app/src/main/java/amazingme/activities/app/UserProfileActivity.java

```diff
@@ -0,0 +1,30 @@
+package amazingme.activities.app;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.view.View;
+import android.widget.Button;
+
+import com.amazingme.activities.R;
+
+public class UserProfileActivity extends AppCompatActivity {
+
+    private Button backtoRBtn;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_user_profile);
+
+        backtoRBtn = (Button) findViewById(R.id.backtoRBtn);
```

Comment:
let's not use abbreviations (like "R") here. and camel case :) 

I'd rather "backToRegister" button than "backtoR". 

but i think convention for us is looking like just take whatever text is on the button (as camel case) and append Btn to it (so this will really just end up being "backBtn". once again, context will come from the encapsulating class and the setOnClickListener

Reply:
Changed the private variable to backBtn and changed the id to backToRegisterBtn